scripts/rollup/packaging.js

```diff
@@ -170,15 +170,24 @@ function copyBundleIntoNodePackage(packageName, filename, bundleType) {
 function copyNodePackageTemplate(packageName) {
   const from = resolve(`./packages/${packageName}`);
   const to = resolve(`./build/packages/${packageName}`);
-
-  // if the package directory already exists, we skip copying to it
-  if (!fs.existsSync(to) && fs.existsSync(from)) {
-    return asyncCopyTo(from, to).then(() =>
-      asyncCopyTo(resolve('./LICENSE'), `${to}/LICENSE`)
-    );
-  } else {
+  const npmFrom = resolve(`${from}/npm`);
+  if (!fs.existsSync(npmFrom)) {
+    // The package is not meant for npm consumption.
+    return Promise.resolve();
+  }
+  if (fs.existsSync(to)) {
+    // We already created this package (e.g. due to another entry point).
     return Promise.resolve();
   }
+  // TODO: verify that all copied files are either in the "files"
+  // whitelist or implicitly published by npm.
```

Comment:
This is something the release script could do too, if it makes more sense there.